offer-manager-domain/src/main/groovy/life/qbic/business/products/copy/CopyProductOutput.groovy

```diff
@@ -0,0 +1,19 @@
+package life.qbic.business.products.copy
+
+import life.qbic.business.UseCaseFailure
+import life.qbic.datamodel.dtos.business.services.Product
+
+/**
+ * Output interface for the {@link CopyProduct} use case
+ *
+ * @since: 1.0.0
+ *
+ */
+interface CopyProductOutput extends UseCaseFailure{
+
+    /**
+     * A copy of a product has been created
```

Comment:
Please state shortly when this method is called in the process. So was the product already stored or when do I expect to see this method?

Reply:
d80256c